docs/working-drafts/issue-29-erratum.md.adoc

```diff
@@ -0,0 +1,135 @@
+// please start a new sentence on a new line.
+// this facilitates commentary on this document using code-review tools.
+
+= Issue 29 - dfdl:prefixLengthKind Simple Type having DFDL Statement Annotations
+
+
+Sections of the DFDL v1.0 Spec which will require updating include:
+
+## Section 9.5 - Evaluation Order for Statement Annotations
+
+An element (or element reference) having lengthKind 'prefixed' can
+have statements associated with its prefix length type. This is
+discussed below in Section (new sub-section of 9.5).
+
+A new sub-section is added: Evaluation Order for Statement Annotations
+for Elements with dfdl:lengthKind 'prefixed'.
+
+This new sub-section contains the content below.
+
+When an element or element reference has lengthKind 'prefixed' there
+can be additional statement annotations associated with the prefix
+length type.
+
+Definition: the resolved set of statement annotations for the simple
+type that is identified by the dfdl:prefixLengthKind property are
+called the _prefix statement annotations_.
+
+Definition: the resolve set of statement annotations for the nested
+prefix simple type that is identified by the dfdl:prefixLengthKind
+property on a simple type itself having lengthKind 'prefixed', are
+called the _prefix-prefix statement annotations_.
+
+For statement annotations associated with any prefix length, any DFDL
+expressions evaluated cannot refer to the DFDL Infoset using path
+expressions.
+
+The value of the prefix (or prefix-prefix) itself can be referenced
+using the expression ".".
+
+When parsing the value of the "." expression for a prefix (or
+prefix-prefix) quasi-element is as before any adjustment (subtraction)
+from the value based on dfdl:prefixIncludesPrefixLength.
+
+When unparsing the value of the "." expression for a prefix (or
+prefix-prefix) quasi-element is as after any adjustment (addition) to
+the value based on dfdl:prefixIncludesPrefixLength.
+
+The evaluation order of statements associated with prefix lengths is:
+
+1. prefix-prefix statement annotations
+
+2. prefix statement annotations
+
+3. statement annotations for the element itself per the description in
+Section 9.5 above.
+
+The statement annotations for a prefix (or prefix-prefix) are
+evaluated in a similar manner as if they were elements.
+The prefix (or prefix-prefix) length fields can be considered
+_quasi-elements_, keeping in mind that as they do not have element
+declarations these can never be nilled, nor can they have array
+dimension, be optional, be empty or have default values as those
+require XSD attributes or DFDL properties that are expressed only on
+element declarations/references.
+
+This implies that in terms of the rules in section 9.3.2 about
+establishing representation, prefix (or prefix-prefix) quasi-elements
+never have zero length, and always have simple normal representation.
+
+The evaluation order for the prefix (or prefix-prefix) statement annotations is:
```

Comment:
Drop "statement" - the bullets include format annotations

Reply:
Per 9.5 numbered item 2 we refer to the parsing of the element itself by way of processing the format annotation.